beast/physicsmodel/creategrid.py

```diff
@@ -299,7 +300,7 @@ def make_extinguished_grid(
     if isinstance(spec_grid, str):
         ext = spec_grid.split(".")[-1]
         if ext in ["hdf", "hd5", "hdf5"]:
-            g0 = SpectralGrid(spec_grid, backend="hdf")
+            g0 = SpectralGrid(spec_grid, backend="disk")
```

Comment:
It's unclear to me which backend option needs to be used in which case.

Reply:
Check out the new docs in `docs/development_classes.rst`.  Hopefully this answers your question.